File: MDEV-1004-S23-ICE8/MDEV1004-S23-ICE8/Base.lproj/Movie.swift
struct Movie: Codable
{
    let _id: String
    let movieID: Int
    let title: String
    let studio: String
    let genres: [String]
    let directors: [String]
    let writers: [String]
    let actors: [String]
    let year: Int
    let length: String
    let shortDescription: String
    let mpaRating: String
    let criticsRating: Double
    let posterLink: String
}

File: MDEV-1004-S23-ICE10C/MDEV1004S23ICE10c/app/src/main/java/com/example/mdev1004s23ice7c/Movie.kt
package com.example.mdev1004s23ice7c

import java.io.Serializable

data class Movie(
    val _id: String,
    val movieID: Int,
    val title: String,
    val studio: String,
    val genres: List<String>,
    val directors: List<String>,
    val writers: List<String>,
    val actors: List<String>,
    val year: Int,
    val length: String,
    val shortDescription: String,
    val mpaRating: String,
    val criticsRating: Double,
    val posterLink: String
) : Serializable

data class MovieResponse(
    val success: Boolean,
    val msg: String,
    val data: MutableList<Movie>
)

data class MovieResponseWrapper(
    val success: Boolean,
    val msg: String,
    val data: Movie
)

data class MovieDeleteWrapper(
    val success: Boolean,
    val msg: String,
    val data: String
)

data class PaymentWrapper(
    val success: Boolean,
    val msg: String,
    val clientSecret: String,
    val customer: String,
    val publishableKey: String
)